extensions/amp-ad-network-doubleclick-impl/doubleclick-rtc.md

```diff
@@ -17,6 +17,15 @@ AMP Real Time Config (RTC) is a feature of Fast Fetch that allows Publishers to
 
 For instructions on how to set the rtc-config attribute on the amp-ad, refer to [Setting Up RTC-Config](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/rtc-publisher-implementation-guide.md#setting-up-rtc-config) in the Publisher Implementation Guide.
 
+## Available URL Macros
+Doubleclick's RTC implementation has made many macros available for RTC url expansion. Please note that the time to expand the URL is counted against the RTC timeout. Currently available macros are as follows:
+
+* **PAGEVIEWID**: pageViewId for the amp-ad element.
+* **HREF**: window.location.href
+* **TGT**: The JSON stringified targeting parameter extracted from the JSON attribute on the amp-ad element.
+* **ADCID**: The adClientId that corresponds to the given publisher page and DoubleClick. This adCid should be the same as used by AMP Analytics. IMPORTANT NOTE: This is an asynchronous call to use this macro, and can be slow. To impose a strict timeout, can pass an optional timeout. This is done in the url like: `https://www.foo.com/?adcid=ADCID(10)` where the passed timeout is in milliseconds.
+* **ATTR**: Makes various attributes on the amp-ad element available, by calling as ATTR(attribute). Currently available attributes are: height, width, data-slot, data-multi-size, data-multi-size-validation, data-override-width, data-override-height. 
```

Comment:
What gets returned if parameter does not exist?

Reply:
Empty string, added to docs.